components/core/DownloadDaemon.py

```diff
@@ -7,6 +7,7 @@
 from EMail import send_mail
 from DiskMan import *
 from ConfReader import get_conf_reader
+from fileminio import *
```

Comment:
Shall we change this from `fileminio` to `FileServer` or `MinioServer`?

Reply:
sure, `MinioServer` will be better than `fileminio`

Reply:
Few more suggestion - `MinioHelper`, `MinioFSManager`, `FileHandler`, `MinioHandler`, ``